charts/opentelemetry-demo/Chart.lock

```diff
@@ -1,15 +1,15 @@
 dependencies:
 - name: opentelemetry-collector
   repository: https://open-telemetry.github.io/opentelemetry-helm-charts
-  version: 0.40.7
+  version: 0.44.1
```

Comment:
Should we set this to the latest just released?

```suggestion
  version: 0.45.0
```

Reply:
@puckpuck ya I'll get this PR updated.